src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -871,13 +871,14 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 						} else if (calledMethod.Parameters.Count > 2 && calledMethod.Parameters[2].ParameterType.Name == "BindingFlags" && methodParams[3].AsConstInt () != null) {
 							bindingFlags = (BindingFlags) methodParams[3].AsConstInt ();
 						}
+						var caseOption = (bindingFlags & (BindingFlags.IgnoreCase)) == BindingFlags.IgnoreCase ? StringComparison.OrdinalIgnoreCase : StringComparison.Ordinal;
```

Comment:
This will need to assume `IgnoreCase` is set if this is an overload that has binding flags and we couldn't deduce the value statically (`methodParams[3].AsConstInt () == null`).